src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs

```diff
@@ -11,65 +11,70 @@
-        public string GetUserCredentialsEndpoint { get; set; }
-        public string GetUserDataEndpoint { get; set; }
-        public string GetUsersDataEndpoint { get; set; }
-        public string CreateAdminEndpoint { get; set; }
-        public string FindParseEntitiesEndpoint { get; set; }
-        public string CheckUsersExistenceEndpoint { get; set; }
+  public class RabbitMqConfig : BaseRabbitMqConfig
+  {
+    public string CompanyServiceUrl { get; set; }
+    public string GetUserCredentialsEndpoint { get; set; }
+    public string GetUserDataEndpoint { get; set; }
+    public string GetUsersDataEndpoint { get; set; }
+    public string CreateAdminEndpoint { get; set; }
+    public string FindParseEntitiesEndpoint { get; set; }
+    public string CheckUsersExistenceEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IAddImageRequest))]
-        public string AddImageEndpoint { get; set; }
+    // file
 
-        [AutoInjectRequest(typeof(IGetImagesRequest))]
-        public string GetImagesEndpoint { get; set; }
+    [AutoInjectRequest(typeof(IAddImageRequest))]
+    public string AddImageEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IGetDepartmentUserRequest))]
-        public string GetDepartmentUserEndpoint { get; set; }
+    [AutoInjectRequest(typeof(IGetImagesRequest))]
+    public string GetImagesEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IFindDepartmentUsersRequest))]
-        public string FindDepartmentUsersEndpoint { get; set; }
+    // project
 
-        [AutoInjectRequest(typeof(IGetPositionRequest))]
-        public string GetPositionEndpoint { get; set; }
+    [AutoInjectRequest(typeof(IGetProjectsRequest))]
+    public string GetProjectsEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IChangeUserDepartmentRequest))]
-        public string ChangeUserDepartmentEndpoint { get; set; }
+    // message
 
-        [AutoInjectRequest(typeof(IChangeUserPositionRequest))]
-        public string ChangeUserPositionEndpoint { get; set; }
+    [AutoInjectRequest(typeof(ISendEmailRequest))]
+    public string SendEmailEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IGetProjectsRequest))]
-        public string GetProjectsEndpoint { get; set; }
+    // auth
 
-        [AutoInjectRequest(typeof(ISendEmailRequest))]
-        public string SendEmailEndpoint { get; set; }
+    [AutoInjectRequest(typeof(IGetTokenRequest))]
+    public string GetTokenEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IGetTokenRequest))]
-        public string GetTokenEndpoint { get; set; }
+    // search
 
-        [AutoInjectRequest(typeof(ISearchUsersRequest))]
-        public string SearchUsersEndpoint { get; set; }
+    [AutoInjectRequest(typeof(ISearchUsersRequest))]
+    public string SearchUsersEndpoint { get; set; }
 
-        [AutoInjectRequest(typeof(IChangeUserRoleRequest))]
-        public string ChangeUserRoleEndpoint { get; set; }
+    // rights
```

Comment:
do we need this comment?

Reply:
this config is big, so I decide to add this comment for creating structure of config

Reply:
also it is not safety